clang/include/clang/Driver/Options.td

```diff
@@ -6103,6 +6103,12 @@ def fsycl_enable_int_header_diags: Flag<["-"], "fsycl-enable-int-header-diags">,
   HelpText<"Enable diagnostics that require the SYCL integration header.">,
   MarshallingInfoFlag<LangOpts<"SYCLEnableIntHeaderDiags">>;
 
+// FIXME: the option probably needs better name
```

Comment:
How about -fsycl-use-main-file-name ?

Reply:
Switched to it in 0a351d1a1b556e14c1dd679bae13ea3e374631dc, will leave this open for a bit more to see if there are other suggestions

Reply:
Leaving it open for better suggestions is fine by me.